packages/mjml-section/src/index.js

```diff
@@ -55,6 +55,8 @@ export default class MjSection extends BodyComponent {
 
     const fullWidth = this.isFullWidth()
 
+    const borderCollapse = this.hasBorderRadius()
```

Comment:
I think that's a better name for the variable:
```suggestion
    const hasBorderRadius = this.hasBorderRadius()
```